src/CONST.ts

```diff
@@ -3571,7 +3571,6 @@ const CONST = {
     LEGAL_NAMES_CHARACTER_LIMIT: 150,
     LOGIN_CHARACTER_LIMIT: 254,
     CATEGORY_NAME_LIMIT: 256,
```

Comment:
Do we still need this constant? Looks like we're only using it [here](https://github.com/Expensify/App/blob/b59091066b9315ea9d0d2c8b8b5c6a35fb9ffb86/src/components/TextPicker/TextSelectorModal.tsx#L28) now, can we update that to `API_TRANSACTION_CATEGORY_MAX_LENGTH` or not?